go.mod

```diff
@@ -4,6 +4,7 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973 // indirect
 	github.com/blacktear23/go-proxyprotocol v0.0.0-20171102103907-62e368e1c470
+	github.com/ciscoxll/parser v0.0.0-20181120113012-ce0923282643 // indirect
```

Comment:
It should be 

```
replace github.com/pingcap/parser => github.com/ciscoxll/parser v0.0.0-20181120113012-ce0923282643
```

at the end of the file.

Reply:
@tiancaiamao Done